proto/sentry_protos/snuba/v1/endpoint_trace_item_stats.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "sentry_protos/snuba/v1/request_common.proto";
+import "sentry_protos/snuba/v1/trace_item_filter.proto";
+
+message AttributesDistributionRequest {
+  // max_buckets and limit wouldn't apply to other stats types (like totals)
+  uint32 max_buckets = 1;
+  uint32 limit = 2;
+}
+
+message StatsType {
+  oneof type {
+    AttributesDistributionRequest attributes_distribution = 1;
+  }
+}
+
+message TraceItemStatsRequest {
+  RequestMeta meta = 1;
+  // same filter applies to all stats types requested
+  TraceItemFilter filter = 2;
+  repeated StatsType stats_types = 3;
+}
+
+message AttributeDistribution {
+  message Bucket {
+    string label = 1;
+    float value = 2;
+  }
+
+  string aggregation = 1;
+  string attribute_name = 2;
+  repeated Bucket buckets = 3;
+}
+
+message AttributesDistribution {
```

Comment:
I would call that `AttributeDistributions` since it's many `AttributeDistribution`.